docs/content/dagster-cloud/managing-deployments/managing-multiple-projects-and-teams.mdx

```diff
@@ -0,0 +1,433 @@
+---
+title: "Managing multiple projects & teams in Dagster Cloud | Dagster Docs"
+description: "How to set up and manage multiple projects with Dagster Cloud Hybrid deployments."
+---
+
+# Managing multiple projects & teams in Dagster Cloud
+
+In this guide, we'll cover some strategies for managing multiple projects/code bases and teams in a Dagster Cloud account.
+
+---
+
+## Separating code bases
+
+<Note>
+  In this section, repository refers to a version control system, such as Git or
+  Mercurial.
+</Note>
+
+If you want to manage complexity or divide your work into areas of responsibility, consider isolating your code bases into multiple projects with:
+
+- Multiple directories in a single repository, or
+- Multiple repositories
+
+Refer to the following table for more information, including the pros and cons of each approach.
+
+<table
```

Comment:
I put the info you had in the next two sections into this single table. It not only visually condenses the information, it makes it a lot easier for the reader to see and compare all approaches.